sdk/js/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@certusone/wormhole-sdk",
-  "version": "0.9.18",
+  "version": "0.9.19-beta.3",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "@certusone/wormhole-sdk",
-      "version": "0.9.18",
+      "version": "0.9.19-beta.3",
```

Comment:
Beta?  Does that mean this is going to have to change again relatively soon?

Reply:
Changed to 0.9.19